usage/src/com/cloud/usage/UsageSanityChecker.java

```diff
@@ -72,7 +72,9 @@ protected boolean checkItemCountByPstmt(CheckCase checkCase) throws SQLException
         try (PreparedStatement pstmt = conn.prepareStatement(checkCase.sqlTemplate)) {
             if(checkCase.checkId) {
                 pstmt.setInt(1, lastId);
-                pstmt.setInt(2, maxId);
+                if (maxId > 0) {
+                    pstmt.setInt(2, maxId);
+                }
```

Comment:
Hi @jburwell, actually when `maxId = -1` parameter is not set to prepared statement, it is only added when it is found in `readMaxId()`